src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,23 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task that has a deadline.
+ */
+public class Deadline extends duke.Task {
+
+    private LocalDate date;
+
+    public Deadline(String name, String dateTime) {
```

Comment:
Perhaps the parameter should not be named `dateTime` as it might imply that it includes a time?